WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift

```diff
@@ -55,6 +82,11 @@ final class DashboardBlazeCampaignCardCell: DashboardCollectionViewCell {
         }
     }
 
+    @objc private func buttonCreateCampaignTapped() {
+        guard let presentingViewController, let blog else { return }
+        BlazeFlowCoordinator.presentBlaze(in: presentingViewController, source: .dashboardCard, blog: blog)
```

Comment:
TBD: do we want to distinguish between the Blaze Promo card and the Blaze Campaigns card when we track events?
 
Slack ref: p1686302601238269-slack-C04LJFS1G5P

Reply:
Thanks for raising this. It looks like it's going to be one card. I added a note to address it when I get to analytics.